docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,111 @@
+.. _defining_YAML_tags:
+
+UW YAML tags
+============
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW defines the following tags to support use cases not covered by standard tags:
+
+``!float``
+^^^^^^^^^^
+
+Converts the tagged node to a Python ``float`` value. For example:
+
+Given input.yaml:
```

Comment:
```suggestion
Given ``input.yaml``:
```
Only noticed when viewing the rendered HTML docs... Will add suggestions for the other cases.